Swift/Tests/DatabaseTest.swift

```diff
@@ -412,6 +412,27 @@ class DatabaseTest: CBLTestCase {
                                                   "middleName": "Lion"]) // conflicting update
     }
     
+    func testConflictHandlerThrowingException() throws {
+        let doc = createDocument("doc1")
+        doc.setString("Daniel", forKey: "firstName")
+        doc.setString("Tiger", forKey: "lastName")
+        try db.saveDocument(doc)
+        
+        let doc1a = db.document(withID: doc.id)!.toMutable()
+        let doc1b = db.document(withID: doc.id)!.toMutable()
+        doc1a.setString("Scott", forKey: "firstName")
+        try db.saveDocument(doc1a)
+        
+        // conflicting save
+        doc1b.setString("Lion", forKey: "middleName")
+        XCTAssertFalse(try db.saveDocument(doc1b) { (cur, old) -> Bool in
```

Comment:
What does happen to the exception thrown? Crash?

Reply:
It won't crash, unless they force unwrap try like this.`try! db.saveDocument(doc1a) { (doc, old) -> Bool in`

Reply:
We already have a test which gets back the `CBLErrorNotFound` error. 
https://github.com/couchbase/couchbase-lite-ios/blob/0f2cd0edfecffcec1212fa0aa108a9019fced8eb/Swift/Tests/DatabaseTest.swift#L452